src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -58,11 +60,9 @@ public function __construct(LanguageManagerInterface $languageManager, ConfigFac
 
     /**
      * Checks for available translation updates.
-     *
-     * @command locale:check
-     * @aliases locale-check
-     * @validate-module-enabled locale
      */
+    #[CLI\Command(name: 'locale:check', aliases: ['locale-check'])]
```

Comment:
As of today, we are creating constants for command names. See https://github.com/drush-ops/drush/pull/5408/files#diff-1900bf031c5b584de69f3be1f8f258797b6e815cbc99a0e1f81fcbdfdd5071f7R21-R24 for an example. In addition to the commandfile, you can use the constants in tests, and site owners can use them in scripts.